pkg/cmd/repo/edit/edit_test.go

```diff
@@ -201,6 +201,36 @@ func Test_editRun(t *testing.T) {
 					}))
 			},
 		},
+		{
+			name: "enable/disable security and analysis settings",
```

Comment:
This test doesn't give me confidence that the code is behaving as expected. Setting the `SecurityAndAnalysis` struct on `opts` manually here is short-circuiting the flag handling, meaning there's no coverage in the logic that transforms the bool flag to the API struct.

That said, I don't expect this test to change with the refactor I've suggested above. I would expect to see a few tests covering the transformation function I've mentioned above, though, which should shore up the gap in our testing, here 🙂

Reply:
Added unit tests for the transformer function:
* enable all 
* disable all
* enable only one
* disable only one

Do you think these provide enough coverage?

Reply:
Yup! And the new set of tests are great! Thanks 🙌